pipes/config.yaml

```diff
@@ -192,6 +196,10 @@ mafft_maxiters: 1000
 # See: https://github.com/trinityrnaseq/trinityrnaseq/wiki/Trinity-FAQ#ques_why_so_many_transcripts
 trinity_n_reads: 250000
 
+# The number of reads to be subsampled from input bam files
+# and used as input to assembly via SPAdes
+spades_n_reads: 250000
```

Comment:
this should be higher, right? like 10000000?

Reply:
yes, thanks for catching.  this is why the defaults should ideally only be in one place (right now they're in three :) )